client/app/components/onboardingGuide/GuideInvitationPage.js

```diff
@@ -1,26 +1,29 @@
 import { PropTypes } from 'react';
 import r, { div, h2, p, img, a } from 'r-dom';
 import css from './styles.scss';
+import { t } from '../../utils/i18n';
 
 import GuideBackToTodoLink from './GuideBackToTodoLink';
 
 const GuideInvitationPage = (props) => {
-  const { changePage, initialPath, t, pageData, infoIcon } = props;
+  const { changePage, initialPath, pageData, infoIcon } = props;
 
   return div({ className: 'container' }, [
-    r(GuideBackToTodoLink, { changePage, initialPath, t }),
-    h2({ className: css.title }, t('title')),
-    p({
-      className: css.description,
-      dangerouslySetInnerHTML: { __html: t('description') }, // eslint-disable-line react/no-danger }
-    }),
+    r(GuideBackToTodoLink, { changePage, initialPath }),
+    h2({ className: css.title }, t('web.admin.onboarding.guide.invitation.title')),
+    p({ className: css.description }, t('web.admin.onboarding.guide.invitation.description.content', {
+      preview_link: a({
+        href: '/?big_cover_photo=true',
+        alt: t('web.admin.onboarding.guide.invitation.description.preview_link_alt'),
+        target: '_blank',
```

Comment:
Ok for me if it already was designed like that. Just couldn't see it in the diff since the old link was somewhere else in the translations.